### ISSUE TITLE
map is 0

### ISSUE THREAD

**Issue by Wulingtian** [Carti-97/DINOv3-Mask2former#2]
09/19/2025 16:15:10 - INFO - __main__ - epoch 0: {'map': tensor(0.), 'map_50': tensor(0.), 'map_75': tensor(0.), 'map_small': tensor(0.), 'map_medium': tensor(0.), 'map_large': tensor(0.), 'mar_1': tensor(0.), 'mar_10': tensor(0.), 'mar_100': tensor(0.), 'mar_small': tensor(0.), 'mar_medium': tensor(0.), 'mar_large': tensor(0.), 'map_per_class': tensor([0., 0., 0., 0., 0., 0., 0.]), 'mar_100_per_class': tensor([0., 0., 0., 0., 0., 0., 0.]), 'classes': tensor([0, 1, 2, 3, 4, 5, 6], dtype=torch.int32)}
09/19/2025 16:15:10 - INFO - __main__ - 🏆 New BEST EPOCH: 0, mAP: 0.0000
`backbone_config` is `None`. Initializing the config with the default `Swin` backbone.
`backbone_config` is `None`. Initializing the config with the default `Swin` backbone.
`backbone_config` is `None`. Initializing the config with the default `Swin` backbone.
Configuration saved in mask2former-swin-small-finetuned\epoch_0\config.json
Model weights saved in mask2former-swin-small-finetuned\epoch_0\model.safetensors
Image processor saved in mask2former-swin-small-finetuned\epoch_0\preprocessor_config.json

**Comment by Wulingtian** [comment]
> The height and width of the input images must be a multiple of 16.

{
    "model_name_or_path": "facebook/mask2former-swin-small-coco-instance",
    "dataset_name": "datasets",
    "output_dir": "mask2former-swin-small-finetuned",
    "do_reduce_labels": true,
    "image_height": 512,
    "image_width": 512,
    "num_train_epochs": 25,
    "learning_rate": 5e-5,
    "lr_scheduler_type": "cosine",
    "num_warmup_steps": 500,
    "per_device_train_batch_size": 2,
    "per_device_eval_batch_size": 4,
    "gradient_accumulation_steps": 4,
    "dataloader_num_workers": 4,
    "checkpointing_steps": "epoch",
    "push_to_hub": false,
    "weight_decay": 1e-4,
    "adam_beta1": 0.9,
    "adam_beta2": 0.999,
    "adam_epsilon": 1e-8,
    "use_data_augmentation": true
}

**Comment by Wulingtian** [comment]
> What I mean is that the dimensions of the images in the original dataset should be a multiple of 16. I've tested this on a dataset where all images have the same dimensions. You shouldn't adjust the image size in the configuration; the images need to be fed into the model with their original sizes. Otherwise, an error occurs because the mask size and the image size do not match.

My original image width and height are 1280 and 720, both are multiples of 16, but the map is still 0.
